### DISCUSSION TITLE
Can QField use attachment directories that are outside the project directory?

### DISCUSSION THREAD

**Discussion by AveryAndrews** [opengisch/QField#3726] [Q&A]
I want to do this to avoid uploading & downloading large numbers of pictures that haven't changed between importing the project and synchronizing it back, by cable. The 'attachment directories' section of the QField tab for Project Configuration can be made to produce a sister directory to the project export folder called something like 'Photos', by putting '../Photos' in it, but I haven't managed to get QField to put pictures there, even if it is copied as a sister folder to the QField project folder (all inside .../Android/data/ch.opengis.qfield).

The idea being that for synchronization and then re-export to the phone, the copying of the attachment folders would be told to ignore files with the same name that are already present.

**Comment by suricactus** [comment]
The answer to your question is no, QField cannot use directories that are outside the project directory for attachments.

However, this feature in QFieldSync is going to improve the situation https://github.com/opengisch/qfieldsync also for you, I guess?

**Reply by AveryAndrews** [reply]
Thanks!  But I'm not sure which feature.  Using an app such as one of the autosync ones to transfer the files between the project on the phone and the import and export folders used by QFieldSync might be the solution.

**Reply by suricactus** [reply]
Ah, sorry, I copied the wrong link. I mean this one here: https://github.com/opengisch/qfieldsync/pull/478
